pandas-stubs/core/frame.pyi

```diff
@@ -270,29 +270,33 @@ class DataFrame(NDFrame, OpsMixin):
         self,
         orient: Literal["dict", "list", "series", "split", "tight", "index"],
         into: Mapping | type[Mapping],
+        index: bool = True,
```

Comment:
This is going to have to split to multiple overloads.  First, you should say `index : bool Literal[True]` which is possible for all the values of `orient`.  But you have to split out `"split"` and `"tight"` that have `index: bool = ...`

Reply:
> This is going to have to split to multiple overloads. First, you should say `index : bool Literal[True]` which is possible for all the values of `orient`.

Do you mean `index : bool | Literal[True]` but Literal[True] is redundant in a union with bool

Reply:
Sorry, that was a typo.  I mean `index: Literal[True]`